util/fqdn.go

```diff
@@ -70,3 +72,7 @@ func DNSNamesExist(cert *x509.Certificate) bool {
 		return true
 	}
 }
+
+func ICANNPublicSuffixParse(domain string) (*publicsuffix.DomainName, error) {
+	return publicsuffix.ParseFromListWithOptions(publicsuffix.DefaultList, domain, &publicsuffix.FindOptions{IgnorePrivate: true})
```

Comment:
We need to add all the options or the library will return a segfault, so instead do this:
`return publicsuffix.ParseFromListWithOptions(publicsuffix.DefaultList, domain, &publicsuffix.FindOptions{IgnorePrivate: true, DefaultRule: publicsuffix.DefaultRule})`